Specs/Number/Japanese/NumberModel.json

```diff
@@ -1544,7071 +1542,3968 @@
   },
   {
     "Input": "私の家は四川にあります",
-    "Comment": "PendingValidation",
+    "NotSupported": "dotnet, javascript, python, java",
     "Results": []
   },
   {
-    "Input": "答えは１９２です。",
-    "IgnoreResolution": true,
-    "NotSupportedByDesign": "javascript, java, python",
+    "Input": "200.2",
```

Comment:
Do not remove existing specs. Any new specs should be added at the end of the file.